android-components/components/browser/state/src/main/java/mozilla/components/browser/state/state/CustomTabSessionState.kt

```diff
@@ -34,6 +34,7 @@ data class CustomTabSessionState(
     override val source: SessionState.Source = SessionState.Source.Internal.CustomTab,
     override val restored: Boolean = false,
     override val cookieBanner: CookieBannerHandlingStatus = CookieBannerHandlingStatus.NO_DETECTED,
+    override val reportSiteDomainSnackBarVisibility: Boolean = false,
```

Comment:
Same as https://github.com/mozilla-mobile/firefox-android/pull/1298/files#r1145301607

Reply:
This UI feature should be implemented at the application level, as not all apps will have this UI functionality.